inc/parser/xhtml.php

```diff
@@ -59,16 +59,24 @@ class Doku_Renderer_xhtml extends Doku_Renderer {
     /**
      * Register a new edit section range
      *
-     * @param string $type   The section type identifier
-     * @param string $title  The section title
      * @param int    $start  The byte position for the edit start
+     * @param array  $data   Assoziative array with section data:
+     *                       Key 'name': the section name/title
+     *                       Key 'target': the target for the section edit,
+     *                                     e.g. 'section' or 'table'
+     *                       Key 'hid': header id
+     *                       Key 'codeblockOffset': actual code block index
+     *                       Key 'start': reserved, do not use
+     *                       Key 'range': reserved, do not use
      * @return string  A marker class for the starting HTML element
      *
      * @author Adrian Lang <lang@cosmocode.de>
      */
-    public function startSectionEdit($start, $type, $title = null, $hid = null) {
-        $this->sectionedits[] = array(++$this->lastsecid, $start, $type, $title, $hid);
-        return 'sectionedit'.$this->lastsecid;
+    public function startSectionEdit($start, $data) {
```

Comment:
we probably need to check if $data is an array - old plugins may supply a type instead

Reply:
Ok, should we output a ```msg()``` in such a case? Or a ```dbg()``` message?

Reply:
let's do a msg()

Reply:
Done.

Reply:
I'm currently trying to fix the Wrap plugin which is affected by this change. How can that be done in a backwards-compatible way?

Reply:
@selfthinker can you open a new ticket or mailing list discussion. Comments on a already closed PR get lost too easily.

To answer your question, you could inspect the SECEDIT_PATTERN definition maybe?

Reply:
@micgro42 fixed it and made it backwards compatible, and I opened a new issue for missing documentation (#2290).